src/com/lightdev/app/shtm/ImagePreview.java

```diff
@@ -147,8 +147,8 @@ public void paint(final Graphics g) {
     public int getScale() {
         int scale = 100;
         if (pic != null) {
-            int vPct = 100;
-            int hPct = 100;
+            int vPct;
+            int hPct;
             final Dimension ps = getPreferredSize();
```

Comment:
Some value should always be assigned at where the variable is declared. But you can declare the variable later when you can calculate its initial value.